context/app/static/js/components/entity-search/SearchWrapper/utils.js

```diff
@@ -220,6 +220,8 @@ function createSearchkitFacet({ field, identifier, label, type, ...rest }) {
   });
 }
 
+const defaultFields = createField({ fieldName: 'mapped_data_access_level', label: 'mapped_data_access_level' });
```

Comment:
Do we still need this? I remember Kevin and I introduced this at some point, but later moved towards making a request with `useDatasetAccessLevel`.

Reply:
Thanks for the context - I initially left this intact since I wasn't familiar with what this was doing, and have removed it after testing that it doesn't impact functionality.